core/embed/rust/Cargo.toml

```diff
@@ -69,3 +82,10 @@ version = "1.0.69"
 [build-dependencies.glob]
 optional = true
 version = "0.3.0"
+
+[patch.crates-io.blake2b_simd]
+path = "./blake2b_hal"
+
+[patch.crates-io.pasta_curves]
+git = "https://github.com/jarys/pasta_curves"
+rev = "a4f755013aad344982383c9f5af362697d928325"
```

Comment:
Sorry for the random dive-bomb comment, but I'm laughing at myself for how long (30+ minutes) it took me to figure out why this is pointing to a rev instead of a version. You have it very nicely documented in the PR description (with links to issues submitted to upstream!), and because I don't RTFM, I'm hunting down this commit, seeing what branch it is in, checking the GitHub Network Graph to see if that is on a popular fork, etc.

I was able to guess that commenting out the inline annotation was to reduce compiled size, but couldn't figure out the other change to `hash_to_curve`. I've only barely tinkered with rust, so that probably doesn't help any.

Hopefully my little adventure makes you laugh!

Reply:
:smile: yeah . pr description seems to be long , but worths it